model/host/db.go

```diff
@@ -483,3 +484,54 @@ func inactiveHostCountPipeline() []bson.M {
 		},
 	}
 }
+
+// struct for storing Id and FinishTime pairs
+type FinishTime struct {
```

Comment:
Nit: Public types should follow Godoc conventions. "FinishTime..." https://blog.golang.org/godoc-documenting-go-code